src/main/java/log/charter/song/configs/Tuning.java

```diff
@@ -145,6 +145,19 @@ public Tuning(final Tuning other) {
 		tuning = Arrays.copyOf(other.tuning, other.tuning.length);
 	}
 
+	public Tuning(int[] gp_tuning, final int capo) {
```

Comment:
should be made into static method so it can be named and not collide

Reply:
See if update is an improvement